spec/indexers/descriptive_metadata/event_contributor_publisher_spec.rb

```diff
@@ -699,13 +699,13 @@
                             }
                           ]
                         }
-                      ],
-                      role: [
-                        {
-                          value: 'publisher'
-                        }
                       ]
                     }
+                  ],
+                  role: [
```

Comment:
The role attribute was in the wrong position. Arcadia confirmed that it should be a sibling of `name`.